code/modules/mob/living/simple_mob/subtypes/animal/pets/cat.dm

```diff
@@ -0,0 +1,136 @@
+/mob/living/simple_mob/animal/passive/cat
+	name = "cat"
+	desc = "A domesticated, feline pet. Has a tendency to adopt crewmembers."
+	tt_desc = "E Felis silvestris catus"
+	icon_state = "cat2"
+	item_state = "cat2"
+	icon_living = "cat2"
+	icon_dead = "cat2_dead"
+	icon_rest = "cat2_rest"
+
+	movement_cooldown = 0.5 SECONDS
+
+	see_in_dark = 6 // Not sure if this actually works.
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "kicks"
+
+	holder_type = /obj/item/weapon/holder/cat
+	mob_size = MOB_SMALL
+
+	has_langs = list("Cat")
+
+	var/mob/living/friend = null // Our best pal, who we'll follow. Meow.
+
+/mob/living/simple_mob/animal/passive/cat/handle_special()
+	if(!stat && prob(2)) // spooky
+		var/mob/observer/dead/spook = locate() in range(src, 5)
+		if(spook)
+			var/turf/T = get_turf(spook)
+			var/list/visible = list()
+			for(var/obj/O in T.contents)
+				if(!O.invisibility && O.name)
+					visible += O
+			if(visible.len)
+				var/atom/A = pick(visible)
+				visible_emote("suddenly stops and stares at something unseen[istype(A) ? " near [A]":""].")
+
+// Instakills mice.
+/mob/living/simple_mob/animal/passive/cat/apply_melee_effects(var/atom/A)
+	if(ismouse(A))
+		var/mob/living/simple_mob/animal/passive/mouse/mouse = A
+		if(mouse.getMaxHealth() > 20) // In case a badmin makes giant mice or something.

```

Comment:
Is this supposed to be more than 20, or less than?

Reply:
Should be correct now, goal was so mice wouldn't be instakilled if they had more than 20 max health.